src/main/java/com/springbootassignment/cruddemo/entity/Course.java

```diff
@@ -0,0 +1,75 @@
+package com.springbootassignment.cruddemo.entity;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "course")
+public class Course {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
+
+    @Column(name = "title")
+    private String title;
+
+    @Column(name = "course_id")
+    private Integer course_id;
+
+
+    @ManyToMany(mappedBy = "courses")
+    private List<Student> students;
+
+    public Course() {
+    }
+
+    public Course(int id, String title) {
+        this.id = id;
+        this.title = title;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
```

Comment:
Use Lombok spring boot dependency and add that above as a annotation.
Pls refer [here](https://springjava.com/spring-boot/use-lombok-in-spring-boot)